src/debug/debugManager.ts

```diff
@@ -1,19 +1,16 @@
 import * as vscode from 'vscode'
-import type { TestRunner } from '../runner/runner'
+import getPort from 'get-port'
+import { getConfig } from '../config'
+import { log } from '../log'
 import type { VitestFolderAPI } from '../api'
-import { showVitestError } from '../utils'
-
-interface VitestDebugConfig {
-  request: vscode.TestRunRequest
-  token: vscode.CancellationToken
-  runner: TestRunner
-  api: VitestFolderAPI
-}
 
 export class TestDebugManager extends vscode.Disposable {
   private disposables: vscode.Disposable[] = []
   private sessions = new Set<vscode.DebugSession>()
-  private configurations = new Map<string, VitestDebugConfig>()
+  private port: number | undefined
+  private address: string | undefined
```

Comment:
```suggestion
  private port?: number
  private address?: string
```

Reply:
I prefer explicit `undefined`, it's not optional, we defined it as undefined

Reply:
Fair enough 🙂